### PR TITLE
Set license files and headers for bitcoin.org's content

### PR DESCRIPTION
**Pull request by ghost1542**
The license for the content on bitcoin.org needs more clarity.

Although the MIT license was mentioned in the footer of the website since the first commit, the website didn't provide licensing information in the usual locations (file headers or COPYING files), except for the libraries and files under different copyright owners or licenses than MIT.

This pull request fixes that issue while carefully avoiding ambiguity or conflicts between multiple licenses with a per-file or per-folder licensing. For this reason, all images under img/ have been moved to a subfolder (I should have carefully tested all layouts with this change).

In the absence of critical feedback, this pull request will be merged on November 28th.

A list of files and folders under a different, or slightly different license or copyright owner:

/_less/normalize.less
/csshover.htc
/font/ubuntu
/font/droidnaskh
/js/leaflet
/js/leaflet-markercluster
/img/flags
/img/brand
/img/wallets
/img/screenshots
/img/faq
/img/innovation
/img/press
/img/os

### CODE REVIEW

**Comment by btcdrak** [comment]
In order for a license change to occur you need the permission of each contributor. A random check of  files has shown several files with more contributors than @saivann and @harding 

You cannot apply license to files with mutliple contributors since each author owns their copyright, you need individual permission. Also MIT does not cover documentation and designs, the wording specifically covers "software".

This particular PR is in fact illegal.

**Comment by harding** [comment]
@btcdrak several contributors were contacted via email by @saivann and asked about this; I was CC'd on some messages.  Every single person I saw respond said that they were under the impression that the site was already MIT licensed because, *as the PR message says*, the site footer has always said "Released under the MIT license".

You can verify this yourself: here's the initial commit: https://github.com/bitcoin-dot-org/bitcoin.org/commit/cb7f7afb93e3de87bcaaf5508815594e70f4a188

This was not a license change.

**Comment by btcdrak** [comment]
Specifically where was the license defined in the first place? You realise the repo needed the license text in the repository as well as MIT headers on each file to in fact license them. 

All that appears in the initial source is an MIT notice on the footer template and it's not even clear what the MIT notice refers to, "the project" seems to refer more to the software than the site remembering at the time the site was created there was only bitcoin core.

**Comment by harding** [comment]
@btcdrak 

> You realise the repo needed the license text in the repository as well as MIT headers on each file to in fact license them.

Yes, which is why this PR exists.  We who contributed content believed we were contributing under the MIT license, but @saivann realized that we were not complying with the terms of the license.  So now we comply with it.

You seem to be under the assumption that because we previously didn't fully comply with the license, we're not allowed to fully comply with the license.

**Comment by btcdrak** [comment]
> You seem to be under the assumption that because we previously didn't fully comply with the license, we're not allowed to fully comply with the license.

No, I am saying it isnt clear that all contributors of every time retroactively licensed has the permission of every contributor who touched the file.

**Comment by harding** [comment]
@btcdrak I can't parse your sentence above.   Could you rephrase?

**Comment by btcdrak** [comment]
golly. what happened there...? I edited my post.

**Comment by harding** [comment]
@btcdrak as posted earlier, several people were contacted about this, and in all the emails I saw, they all agreed that they had contributed under the belief that their content was MIT licensed.  If you think there's someone out there who believes that they contributed under a different license, please have them contact @saivann or myself.

**Comment by saivann** [comment]
@btcdrak For the record, I hired a lawyer when I was working on that issue to make sure things were done correctly.
